src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -18,13 +18,13 @@ namespace PublicApiGenerator
 {
     public static class ApiGenerator
     {
-        static readonly string[] defaultWhitelistedNamespacePrefixes = new string[0];
-
-        public static string GeneratePublicApi(Assembly assembly, Type[]? includeTypes = null, bool shouldIncludeAssemblyAttributes = true, string[]? whitelistedNamespacePrefixes = null, string[]? excludeAttributes = null)
+        public static string GeneratePublicApi(Assembly assembly, ApiGeneratorOptions? options = null)
```

Comment:
Let's make it extension method. It will also be convenient to make such an extension method on a `Type`. In fact the `ApiGeneratorTestsBase` class suffers from the lack of this extension :)

Reply:
I'll pull it in with the options type. Happy to accept PRs to further improve this